tests/CMakeLists.txt

```diff
@@ -106,35 +106,28 @@ ENDIF()
 set(AlbanyPath                         ${Albany_BINARY_DIR}/src/Albany)
 set(AlbanyAnalysisPath                 ${Albany_BINARY_DIR}/src/AlbanyAnalysis)
 
-IF (ALBANY_MPI)
-  set(SerialAlbany.exe                 ${SERIAL_CALL} ${AlbanyPath})
-  set(SerialAlbanyAnalysis.exe         ${SERIAL_CALL} ${AlbanyAnalysisPath})
-  # Do not test on greater than Trilinos_MPI_EXEC_MAX_NUMPROCS configured in Trilinos build
-  # or explicitly given at Albany configure time -D ALBANY_MPI_EXEC_MAX_NUMPROCS
-  IF(DEFINED MPIMNP)
-    IF(MPIMNP LESS 8)
-      SET(MAX_MPI_RANKS ${MPIMNP})
-    ELSE(MPIMNP LESS 8)
-      SET(MAX_MPI_RANKS 8)
-    ENDIF(MPIMNP LESS 8)
-  ELSE(DEFINED MPIMNP)
-      SET(MAX_MPI_RANKS 4)
-  ENDIF(DEFINED MPIMNP)
-  IF (ALBANY_ENABLE_CUDA) 
-    set(Albany.exe                     ${PARALLEL_CALL} ${AlbanyPath} ${KOKKOS_NDEVICES})
-    set(Albany8.exe                    ${MPIEX} ${MPIPRE} ${MPINPF} ${MAX_MPI_RANKS} ${MPIPOST} ${AlbanyPath} ${KOKKOS_NDEVICES})
-    set(AlbanyAnalysis.exe             ${PARALLEL_CALL} ${AlbanyAnalysisPath} ${KOKKOS_NDEVICES})
-  ELSE() 
-    set(Albany.exe                     ${PARALLEL_CALL} ${AlbanyPath})
-    set(Albany8.exe                    ${MPIEX} ${MPIPRE} ${MPINPF} ${MAX_MPI_RANKS} ${MPIPOST} ${AlbanyPath})
-    set(AlbanyAnalysis.exe             ${PARALLEL_CALL} ${AlbanyAnalysisPath})
-  ENDIF() 
-ELSE()
-  set(SerialAlbany.exe                 ${AlbanyPath})
-  set(SerialAlbanyAnalysis.exe         ${AlbanyAnalysisPath})
-  set(Albany.exe                       ${AlbanyPath})
-  set(AlbanyAnalysis.exe               ${AlbanyAnalysisPath})
-ENDIF()
+set(SerialAlbany.exe                 ${SERIAL_CALL} ${AlbanyPath})
```

Comment:
Question: why is our serial call `mpirun -np 1 Albany` rather than just `Albany`? The mpirun does not add anything here, besides (at most) some freedom when it comes to binding the forked thread to a resource. Other than that, it is the same as running the raw executable itself...

It seems a bit of an overcomplication, with associated reading/parsing slowdown, and no tangible added value.